_aggregations/metric/extended-stats.md

```diff
@@ -9,8 +9,41 @@ redirect_from:
 
 # Extended stats aggregations
 
-The `extended_stats` aggregation is an extended version of the [`stats`]({{site.url}}{{site.baseurl}}/query-dsl/aggregations/metric/stats/) aggregation. Apart from including basic stats, `extended_stats` also returns stats such as `sum_of_squares`, `variance`, and `std_deviation`.
-The following example returns extended stats for `taxful_total_price`:
+The `extended_stats` aggregation is a more comprehensive version of the [`stats`]({{site.url}}{{site.baseurl}}/query-dsl/aggregations/metric/stats/) aggregation. As well as the basic statistical measures provided by stats, `extended_stats` calculates:
+
+- Sum of squares
+- Variance
+- Population variance
+- Sampling variance
+- Standard deviation
+- Population standard deviation
+- Sampling standard deviation
+- Standard deviation bounds:
+  - Upper
+  - Lower
+  - Population upper
+  - Population lower
+  - Sampling upper
+  - Sampling lower
+
+The standard deviation and variance are population statistics; they are always equal to the population standard deviation and variance, respectively.
+
+The `std_deviation_bounds` object defines a range that spans the specified number of standard deviations above and below the mean (default is two standard deviations). This object is always included in the output but is meaningful only for normally distributed data. Before interpreting these values, verify that your dataset follows a normal distribution.
+
+## Parameters
+
+The `extended_stats` aggregation takes the following parameters.
+
+| Parameter | Required/Optional | Data type             | Description |
+| :--       | :--               | :--                   | :--         |
+| `field`   | Required          | String                | The name of the field for which the extended stats are returned. |
+| `sigma`   | Optional          | Double (non-negative) | The number of standard deviations above and below the mean used to calculate the `std_deviation_bounds` interval. Default is `2`. |
+| `missing` | Optional          | Numeric        | The value to assign missing instances of the field. If not given, documents with missing values are omitted from the extended stats. |
+
+## Example
+
+This example returns extended stats for `taxful_total_price` in the e-commerce sample data:
+
```

Comment:
Line 45: "data" => "dataset"?

Reply:
yes

Reply:
@dwelsch-esi Let's add for clarity, then.